task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,7 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+    return num % 15 === 0 ? "FizzBuzz" : num % 5 === 0 ? "Buzz" : num % 3 === 0 ? "Fizz" : num;
```

Comment:
Please, avoid this code style, if-else is more readable than multiple conditional operators ? :